java/vector/src/main/java/org/apache/arrow/vector/ipc/message/MessageSerializer.java

```diff
@@ -576,4 +601,5 @@ public static ArrowBuf readMessageBody(ReadChannel in, int bodyLength, BufferAll
     }
     return bodyBuffer;
   }
+
```

Comment:
was this intentional?

Reply:
no, or at least its been long enough that I don't remember.